developer_manual/app_publishing_maintenance/upgrade-guide.rst

```diff
@@ -79,11 +79,6 @@ The original `PSR-0` standard was deprecated in 2014 and therefore the support f
 .. _`PSR-0`: https://www.php-fig.org/psr/psr-0/
 .. _`PSR-4`: https://www.php-fig.org/psr/psr-4/
 
-Last version with database.xml support and migration
```

Comment:
could you please revert this change? This is the upgrade section for people who update their app for Nextcloud 21. This information is still true. Only in 22 this was dropped.

Reply:
instead open a new section for 22 and document that this is now gone.

Reply:
can do, but I'll have to remove the link because the section about the occ command has been removed.
not sure if we should create a link between different versions of the documentation ?!

Reply:
Just drop the link. Developers should be able to figure out the rest :)

Reply:
PR here: https://github.com/nextcloud/documentation/pull/6874